chart/kubeapps/templates/kubeapps-frontend-deployment.yaml

```diff
@@ -70,6 +70,7 @@ spec:
             - -skip-auth-regex=^\/config\.json$
             - -skip-auth-regex=^\/favicon.*\.png$
             - -skip-auth-regex=^\/static\/
+            - -skip-auth-regex=^\/$
             {{- range .Values.authProxy.additionalFlags }}
```

Comment:
do we need to skip the `/login` route as well?

Reply:
No - at least, not currently, because we're using a hash-based url scheme. So the `/login` route is actually `/#/login`, which as far as oauth2proxy is concerned, is the `/` path (like all the app routes), even on a refresh.